### PR TITLE
Update servicedef and policy for new IpCidrMatchers

### PR DESCRIPTION
**Pull request by Bongani**

### CODE REVIEW

File: airlock-dev-apache-ranger/resources/servicedef/ranger-servicedef-s3.json [hunk 1]
```diff
@@ -103,10 +103,18 @@
     [
       {
         "itemId": 1,
-        "name": "cidr",
-        "label": "IP address within cidr range?",
-        "description": "Ip address within cidr range?",
-        "evaluator": "com.ing.wbaa.ranger.plugin.conditionevaluator.IpCidrMatcher",
+        "name": "cidrAllIPs",
```
Comment (kr7ysztof) [inline airlock-dev-apache-ranger/resources/servicedef/ranger-servicedef-s3.json]: wouldn't be the name misleading?  eg `cidrAllIPs:192.0.0.0/24,168.0.0.0/24` The **All** could be related to ips written in ranger not the user ips we want to check (???). 
```suggestion
        "name": "cidrAllUserIPs",
```
unfortunately - the truth is that the longer the name is the worse :( so if it is no misleading for you just leave it.

File: airlock-dev-apache-ranger/resources/servicedef/ranger-servicedef-s3.json [hunk 2]
```diff
@@ -103,10 +103,18 @@
...
+        "label": "All IP addresses within cidr range?",
+        "description": "All IP addresses within cidr range?",
+        "evaluator": "com.ing.wbaa.ranger.plugin.conditionevaluator.AllIpCidrMatcher",
+        "evaluatorOptions": {}
+      },
+      {
+        "itemId": 2,
+        "name": "cidrAnyIPs",
```
Comment (kr7ysztof) [inline airlock-dev-apache-ranger/resources/servicedef/ranger-servicedef-s3.json]:
```suggestion
        "name": "cidrAnyUserIPs",
```

File: airlock-dev-apache-ranger/resources/servicedef/ranger-servicedef-s3.json (hunk 1 above)
Comment (bolkedebruin) [reply airlock-dev-apache-ranger/resources/servicedef/ranger-servicedef-s3.json]: Or CidrRequireAllIps

File: airlock-dev-apache-ranger/resources/servicedef/ranger-servicedef-s3.json (hunk 2 above)
Comment (bolkedebruin) [reply airlock-dev-apache-ranger/resources/servicedef/ranger-servicedef-s3.json]: CidrRequireAnyIps

File: airlock-dev-apache-ranger/resources/servicedef/ranger-servicedef-s3.json (hunk 1 above)
Comment (WalkerTR) [reply airlock-dev-apache-ranger/resources/servicedef/ranger-servicedef-s3.json]: CidrRequireAllIps doesn't really solve the problem IMHO. I used the name suggested by Krzysztof and updated the description as well.

**Review by kr7ysztof — APPROVED** [review]

**Pull request merged**